### PR TITLE
add hack font

### PR DESCRIPTION
**Pull request by sofuture**
add package for [hack](http://sourcefoundry.org/hack/) font

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -6272,8 +6272,8 @@ let
   freealut = callPackage ../development/libraries/freealut { };
 
   freeglut = callPackage ../development/libraries/freeglut { };
-  
-  freenect = callPackage ../development/libraries/freenect { 
+
+  freenect = callPackage ../development/libraries/freenect {
```
Comment (sofuture) [inline pkgs/top-level/all-packages.nix]: editor cleaned up whitespace, unrelated

**Comment by jagajaga** [comment]
Please rename commit's names following our guide in `CONTRIBUTING.md`.

File: pkgs/data/fonts/hack-ttf/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{stdenv, fetchurl, unzip}:
+
+stdenv.mkDerivation rec {
+  name = "hack-ttf";
```
Comment (garbas) [inline pkgs/data/fonts/hack-ttf/default.nix]: version is missing in the name

File: pkgs/data/fonts/hack-ttf/default.nix [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+  installPhase = ''
+    unzip ${src}
+    mkdir -p $out/share/fonts/truetype
+    cp *.ttf $out/share/fonts/truetype
+  '';
+
+  meta = {
+    description = "TrueType version of the Hack font";
+    homepage = http://sourcefoundry.org/hack/;
```
Comment (garbas) [inline pkgs/data/fonts/hack-ttf/default.nix]: would be nice to put yourself as maintainer and also add license

**Comment by jagajaga** [comment]
Why do you close this PR? 
You have to make changes and do force push to your branch.

**Pull request closed without merging**